src/cf-tables/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cf-tables",
-  "version": "1.0.3",
+  "version": "null",
```

Comment:
The cf-tables line in the change log in `canary` was missing its `[MINOR]` label.